scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
Would be nice to have a docstring for this file which describes exactly what it does and how to use it.